packages/debian/postinst

```diff
@@ -20,7 +20,7 @@ adu_diagnostics_conf_file=du-diagnostics-config.json
 adu_log_dir=/var/log/adu
 adu_data_dir=/var/lib/adu
 adu_downloads_dir="$adu_data_dir/downloads"
-adu_shell_dir=/usr/lib/adu
+adu_shell_dir=/usr/bin
```

Comment:
what if customer override shell path in du-config.json? Now, it is out of sync.

Reply:
The assumption is that customers who modify the adu-shell portion are creating a customized agent and likely not installing it through the APT package.